ops/services/okta-app/_var.tf

```diff
@@ -28,6 +28,40 @@ variable "redirect_urls" {
   ]
 }
 
+variable "trusted_origin_preview_urls" {
+  description = "Trusted origin listing for the Okta Preview environment. Data format is [Name, URL, [Scopes]]"
+  default = [
+    ["localhost", "https://localhost.simplereport.gov", ["CORS, REDIRECT"]],
+  ]
+}
+
+variable "trusted_origin_urls" {
+  description = "Trusted origin listing for the Okta Production environment. Data format is [Name, URL, [Scopes]]"
+  default = [
+    ["Localhost (DataHub)", "http://localhost:7071", ["CORS"]],
```

Comment:
Hmm, this gets a little dicey - these URLs are for the ReportStream team, not SimpleReport. The intention here is to strip the trusted domains from Okta after this PR is in, correct? I don't really want to take over their infrastructure like this, so it may be worth only including the SimpleReport trusted origins in our infrastructure and leaving the ReportStream origins as-is in Okta, so they can manage as they see fit.
@jdorothy your thoughts?

Reply:
Oof, I didn't realize that! Yes, I would love to hear guidance on this. I'm happy to make whatever changes are needed.

Reply:
I think my preference would be to not touch the ReportStream URLs and just include the SimpleReport ones here (which I think is just test, stg, prod, and www.prod). 

@MauriceReeves-usds Just flagging this for your attention - we're moving our trusted origins in Okta to be terraform-managed, but planning to leave the ReportStream ones as-is in Okta. Does that work for the ReportStream team?

Reply:
@emmastephenson I've pushed some changes in my most recent commit. If I need to make any further adjustments, please let me know!